src/main/target/common_post.h

```diff
@@ -329,6 +329,12 @@
 #endif
 #endif
 
+#if defined(USE_FLASH_W25N02K) && !defined(USE_FLASH_W25N01G)
+#if !defined(USE_FLASH_W25N01G)
+#define USE_FLASH_W25N01G
```

Comment:
Maybe define common  `USE_FLASH_W25N`? Or some better name that will cover all chips with similar protocol ...

Reply:
Should I just rename the driver to be the w25n driver? I considered doing this.

Reply:
Done. We now have a w25n driver.